app/src/forms/admin/routes.js

```diff
@@ -18,22 +18,160 @@ routes.use(currentUser);
 //
 // Forms
 //
+
+/**
+ * @openapi
+ * /admin/forms:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    description: This endpoint will fetch all forms under the admin access.
+ *    security:
+ *      - openId: []
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/AdminsFormGetEx'
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
+ */
 routes.get('/forms', async (req, res, next) => {
   await controller.listForms(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/forms/{formId}:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    description: This endpoint will fetch the form details(and metadata for versions).
+ *    security:
+ *      - openId: []
+ *    parameters:
+ *      - in: path
+ *        name: formId
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *        description: ID of the form.
+ *        required: true
+ *        example: c6455376-382c-439d-a811-0381a012d696
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/AdminReadFormEx'
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
```

Comment:
If an endpoint takes parameters, it should also respond with a 404 if the parameter(s) are wrong. Can the 404 response be added to all these endpoints?

Reply:
Fix